astropy/nddata/utils.py

```diff
@@ -633,11 +633,11 @@ def __init__(self, data, position, shape=None, side_length=None, wcs=None,
         self.input_position_original = position
         self.shape_input = shape
 
```

Comment:
I also thought, that @larrybradley had already changed this, do you know what happended in between?